specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -2491,13 +2491,25 @@
             "minLength": 1,
             "maxLength": 90
           },
+          {
+            "name": "forceDeletionResourceTypes",
+            "in": "query",
+            "required": false,
+            "type": "string",
```

Comment:
type should be boolean

Reply:
Hi @xccc-msft -- Type in this case is correctly marked as string since this API is called by specifying the types in string format (e.g. ?forceDeletionResourceTypes=Microsoft.Compute/virtualMachines)

Reply:
Yes, this is for different API. I misunderstood this before.